src/grokcore/component/compat.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import martian
+from zope.interface import classImplements
+from zope.interface.interfaces import IInterface
+
+if sys.version_info[0] < 3:
+    from zope.interface import implements
+else:
+
+    class implements(martian.Directive):
+        scope = martian.CLASS
+        store = martian.ONCE
+        default = None
+        
+        def factory(self, *interfaces):
+            for interface in interfaces:
+                if not IInterface.providedBy(interface):
+                    raise Exception('%s is not a interface' % repr(interface))
+            return interfaces
+
+
+    class ClassImplementerGrokker(martian.ClassGrokker):
```

Comment:
Looks good. 

I think it makes sense to keep extending grok.cfg because it specifies other dependencies and versions as well.